src/plugins/Wallet/red-packet-fsm.ts

```diff
@@ -83,12 +83,12 @@ export async function getRedPackets(owned?: boolean) {
 export async function createRedPacket(packet: createRedPacketInit): Promise<{ password: string }> {
     if (packet.send_total < packet.shares) {
         throw new Error('At least [number of red packets] tokens to your red packet.')
-    } else if (packet.shares < 0) {
+    } else if (packet.shares.isLessThan(0) /* packet.shares < 0n */) {
```

Comment:
Does this lib requires compare between bns?

Reply:
doesn't catch your meaning

Reply:
```javascript
packet.shares.isLessThan(new BigNumber(0))
```

Reply:
```typescript
packet.shares.isNegative()
```

fixed